cmd/binder/app/options.go

```diff
@@ -25,37 +29,61 @@ type Options struct {
 	GpuCdiEnabled                        bool
 	VolumeBindingTimeoutSeconds          int
 	GPUSharingEnabled                    bool
-	SchedulerName                        string
 }
 
 func InitOptions() *Options {
 	options := &Options{}
 
 	fs := pflag.CommandLine
 
-	fs.StringVar(&options.ResourceReservePodImage,
-		"resource-reservation-pod-image", "registry/local/kai-scheduler/resource-reservation", "Container image for the resource reservation pod")
+	fs.StringVar(&options.SchedulerName,
+		"scheduler-name", "kai-scheduler",
+		"The scheduler name the workloads are scheduled with")
+	fs.StringVar(&options.ResourceReservationNamespace,
+		"resource-reservation-namespace", "kai-resource-reservation",
+		"Namespace for resource reservation pods")
+	fs.StringVar(&options.ResourceReservationServiceAccount,
+		"resource-reservation-service-account", "kai-resource-reservation",
+		"Service account name for resource reservation pods")
+	fs.StringVar(&options.ResourceReservationPodImage,
+		"resource-reservation-pod-image", "registry/local/kai-scheduler/resource-reservation",
```

Comment:
Where is this path pointing to? registry/local/kai-scheduler/resource-reservation

Reply:
nowhere, it is the default registry we build KAI chart with - it can be loaded to node cache in a kind cluster or replaced with a real registry url if you have one.
This default is being overwritten in the chart:
https://github.com/NVIDIA/KAI-Scheduler/blob/main/deployments/kai-scheduler/templates/services/binder.yaml#L28C4-L28C32